data_fixtures/migrations/0007_unpublish_empty_topic_channels.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.14 on 2024-08-01 20:31
+
+from django.db import migrations
+
+
+def remove_empty_topic_channels(apps, schema_editor):
```

Comment:
Just renamed this 👍

Reply:
> - see other comment about potentially removing the "delete_" plugin methods since they just proxy to unpublish
Discussed this with Shankar in Slack. Other comment seems to have gotten lost somewhere.

Anyway: We can't just remove the `delete_` plugins. They get called when the corresponding hook is fired. We'd have to replace the unpublish + delete hooks with some combined hook, like `resource_before_delete_or_after_unpublish`. A combined hook isn't unreasable—upsert is combined create/update, after all. But leaving as-is for this PR.